### ISSUE TITLE
shell redirection unable to append

### ISSUE THREAD

**Issue by adbennet** [louisdh/terminal#161]
$ echo foo>tmp.txt
$ ls
tmp.txt
$ echo bar>>tmp.txt
$ ls
\>tmp.txt     tmp.txt

I tried looking at the code to track this down, but quickly got lost.

**Comment by holzschu** [comment]
Yes, that is a known issue of `ios_system`. It has been on the todo list for quite some time: https://github.com/holzschu/ios_system/issues/4 
The issue is in lines 1040-1080 of `ios_system.m`. I will look into it at some point.
